Tests/merge_test.py

```diff
@@ -233,8 +233,7 @@ def test_GSUB_merge_with_different_dups_and_no_second_GSUB(self):
 
 		table = self.mergedTable.table
 		self.assertEqual(table, self.table1.table)
-		self.assertTrue(len(captor.records) > 0)
-
+		self.assertTrue(len([r for r in captor.records if r.msg != '']) > 0)
```

Comment:
Doing the warning message matching seems focus too much on the implementation detail not the behavior of the function. Say, the test case will fail if someone else just changes the words in warning message to fix typo or make it more user friendly. In this case I think we should not let the test case fails. Just check there is warning message shown to the user is good enough. Ideas?